### PR TITLE
Feature/hig-743 when you open the conversations app it should always open to

### PR DESCRIPTION
**Pull request by Jungyoonlim**
here's the pr for it!! thanks for asking :)

### CODE REVIEW

**Comment by Jungyoonlim** [comment]
@sam5pell give me 1 second I need to fix some conflicts

**Comment by sam5pell** [comment]
Noticed a few things
1. I couldn't navigate to the current conversation until audio was picked up
2. I couldn't navigate back to a saved conversation after audio was detected

Not sure if this is because this branch is out of date with main or if there's another issue!

**Comment by Jungyoonlim** [comment]
@sam5pell I made some updates, plz check it out! thanks :)

File: components/Conversations/Detail/ConversationDetail.tsx [hunk 1]
```diff
@@ -63,23 +64,33 @@ const useTranscriptionTimer = (isAudioOn: boolean, micActivity: number, saveCurr
...
+const ConversationDetail: React.FC<ConversationDetailProps> = ({ 
+  conversation 
+}) => {
+  const { micActivity, isAudioOn, saveCurrentConversation, conversations } = useConversations()
   const isTranscribing = useTranscriptionTimer(isAudioOn, micActivity, saveCurrentConversation)
 
+  // First priority: Show ActiveConversation when transcribing
+  if (isAudioOn && isTranscribing) {
+    return <ActiveConversation />
+  }
```
Comment (sam5pell) [inline components/Conversations/Detail/ConversationDetail.tsx:76]: @Jungyoonlim this is still preventing me from accessing any older conversations while there's an active conversation. please make sure you test this - keeps coming up in different PRs. let me know if you have any questions about this, we can talk through the logic

**Pull request closed without merging**